spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -39,12 +39,20 @@ def get_value(self, name):
         reason_big = _("The variable is too big to be retrieved")
         reason_not_picklable = _("The variable is not picklable")
         reason_dead = _("The kernel is dead")
-        reason_other = _("An unkown error occurred. Check the console because "
-                         "its contents could have been printed there")
+        reason_other = _("An unkown error occurred. Check the console because"
+                         " its contents could have been printed there")
```

Comment:
```suggestion
        reason_other = _("An unkown error occurred. Check the console because "
                         "its contents could have been printed there")
```

This string was fine as it was.